records_mover/db/unloader.py

```diff
@@ -25,10 +25,15 @@ def unload(self,
     def known_supported_records_formats_for_unload(self) -> List[BaseRecordsFormat]:
         ...
 
+    # TODO: Drop this in name in other PR
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker